aea/helpers/profiling.py

```diff
@@ -148,16 +158,29 @@ def output_profile_data(self) -> None:
         Run time: {data["run_time"]:.6f} seconds
         Cpu time: {data["cpu_time"]:.6f} seconds,
         Cpu/Run time: {100*data["cpu_time"]/data["run_time"]:.6f}%
-        Memory: {data["mem"]:.6f} MB
+        Memory: {data["mem"]:.6f} MB [Peak {data["mem_peak"]:.6f} MB]
         Threads: {data["threads"]['amount']}  {data["threads"]['names']}
         Objects present:
         """
             )
-            + "\n".join([f" * {i}:  {c}" for i, c in data["objects_present"].items()])
+            + "\n".join(
+                [f" * {i} (present):  {c}" for i, c in data["objects_present"].items()]
+            )
             + "\n"
             + """Objects created:\n"""
             + "\n".join(
-                [f" * {i.__name__}:  {c}" for i, c in data["objects_created"].items()]
+                [
+                    f" * {i.__name__} (created):  {c}"
+                    for i, c in data["objects_created"].items()
```

Comment:
I've added those (present) and (created) tags in order to better parse the logs and differenciate between variables.